thor/src/lib.rs

```diff
@@ -596,7 +592,7 @@ impl Channel {
             self.tx_f_body,
             x_self,
             X_other,
-            splice_in,
+            splice,
```

Comment:
I might have messed things up, I just removed a bunch of `_in`s from identifiers and comments around splicing. I was not aware that there was going to be an 'out'.

Reply:
😢